src/test/scala/za/co/absa/commons/io/TempDirectorySpec.scala

```diff
@@ -45,6 +45,21 @@ class TempDirectorySpec extends AnyFlatSpec with Matchers {
     name3 should (startWith("foo") and endWith("bar"))
   }
 
+  it should "return tempDirectory as string" in {
```

Comment:
In this context `it` is a `apply` method, while the scenario is about the `toString` method.
Also the scenario doesn't actually verify the given statement, but other unrelated statements.

Reply:
Fixed in c6675808ba03a8efcc7111030996fdde162b626a.